book/1.json

```diff
@@ -44,27 +45,27 @@
     {"text":"Gen","color":"gold","hoverEvent":{"action":"show_text","value":"Pre-generate the chunks\nwithin the world border\n\nThis may take several minutes!"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 28"}},
     {"text":"\n\n     Shrink Options ","color":"dark_gray"},
     {"storage":"uhc_pack:text","nbt":"Icon.Shrink","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 29"}},
-    {"text":"\nStart","color":"dark_gray"},
+    {"text":"\nStart","color":"dark_gray","hoverEvent":{"action":"show_text","value":"Amount of minutes before the\nborder should start shrinking"}},
```

Comment:
Maybe this should be `The amount of minutes` to be consistent with the duration tooltip